content/en/docs/how-to/plugin-management.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Install and manage notation plugins
+description: Install and manage notation plugins
+weight: 9
+---
+
+Plugins for `notation` provide additional functionality and integration. For example, the [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin allows you to sign a Notation-generated payload with a certificate in [Azure Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/general/overview?wt.mc_id=azurelearn_inproduct_oss_notaryproject).
+
+## Install a plugin
+
+To install a plugin, download the plugin binary, create a `{plugin-name}` directory in `{NOTATION_LIBEXEC}/plugins/`, and add the plugin binary to that directory. The following example downloads and installs the [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin assuming the environment variables are set for the plugin version and install path for your development computer.
```

Comment:
I think it will be good to explain a bit more about what the name of the directory should be. It is dependent on the name of the binary downloaded and it is important to emphasize that.

Reply:
I see that is explained below - maybe add reference. I personally prefer to read this information before I create the folder, though. This way I can go through the example with all the necessary context.

Reply:
I see where you're coming from. I refactored the section to make it easier to follow and understand.